nomad/structs/structs.go

```diff
@@ -3817,6 +3820,129 @@ type TaskEvent struct {
 
 	// GenericSource is the source of a message.
 	GenericSource string
+
+	// DisplayMessage is a human friendly message about the event
```

Comment:
Would you mind making a deprecation notice about the other fields explaining we are transitioning to just these two.